src/components/CommentInput/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import capitalize from '../../utils/capitalize';
+import config from '../../config';
+
+const CommentInput = ({
+  type, name, label, onChange, handleSubmitComment, value, errors, failure, user,
+}) => (
+  <div className="comment-box">
+    <div className="row comment-box__user">
+      <img src={user.image ? user.image : config.DEFAULT_USER_AVATAR} alt="" className="responsive-img circle" />
```

Comment:
I believe we will get an error here if the user is not present. I just got it in a few and it disappeared

Reply:
It happens on the localhost, so you have to refresh to clear the cached data from previous tests
`cmd+shift+R` then test it.